### PR TITLE
feat: style login page

### PR DESCRIPTION
**Pull request by admbtlr**
This is the page that is displayed when not using the UMS. To see it, remove `CAS_SERVER_URL: https://ums-dev.ehealthafrica.org` from your `docker-compose-base.yaml`

### CODE REVIEW

**Comment by obdulia-losantos** [comment]
Nice

image

Logout page

image

Log in again page :cry: 

image

File: aether-ui/aether/ui/settings.py [hunk 1]
```diff
@@ -42,9 +42,10 @@
 }
 
 INSTALLED_APPS += [
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/settings.py]:
```python
INSTALLED_APPS = ['aether.ui'] + INSTALLED_APPS + ['webpack_loader']
```
Do you mean this?

Comment (admbtlr) [reply aether-ui/aether/ui/settings.py]: Yes, true, that's a nicer way of doing it

**Review by obdulia-losantos — APPROVED** [review]

**Comment by obdulia-losantos** [comment]
@adamvert do you want to include this in the next release? :grimacing:

**Comment by admbtlr** [comment]
Ah, thanks for pointing that out!

**Pull request merged**